umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/dif9/additional_attribute.clj

```diff
@@ -0,0 +1,49 @@
+(ns cmr.umm-spec.test.xml-to-umm-mappings.dif9.additional-attribute
+  (:require
+   [clojure.java.io :as io]
+   [clojure.test :refer :all]
+   [cmr.umm-spec.xml-to-umm-mappings.dif9.additional-attribute :as aa]))
+
+(defn- additional-attributes-test-file
+  "Reads in the test file for these tests."
+  []
+  (slurp (io/resource "example-data/dif/C1214305813-AU_AADC.xml")))
+
+(def expected-additional-attributes
+  "This is the normal expected value for most of the tests."
+  '({:Group "gov.nasa.gsfc.gcmd.additionalattribute",
```

Comment:
I think we always use vectors rather than list literals.

Reply:
fixed